app/controllers/api/v1/projects_controller.rb

```diff
@@ -33,7 +33,6 @@ class Api::V1::ProjectsController < Api::ApiController
                  :updated_at].freeze
 
   before_action :filter_by_tags, only: :index
-  before_action :downcase_slug, only: :index
```

Comment:
oh heck, forgot about filtering by slug in the same commit I added filtering by slug. I'll add this back.